.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
             ${{ runner.os }}-target-lint-
       - run: cargo clippy --workspace --all-targets
       # supported winit versions
-      - run: cargo clippy --manifest-path imgui-winit-support/Cargo.toml --all-features --all-targets
+      - run: cargo clippy --manifest-path imgui-winit-support/Cargo.toml --features winit-19 --features winit-20 --features winit-22 --features winit-23/default --features winit-24/default --all-targets
```

Comment:
Ugh, this pretty much sucks. I'd like to find a better way. Note that among other things, you can pass more than one feature at a time, but is there no way to make --all-features work?

Also, please reflect any changes you make here to https://github.com/imgui-rs/imgui-rs/blob/master/xtask/src/main.rs#L38-L61, which is used for local dev (it's not used for CI too so that we get more insight on which operation in particular failed)